quipucords/api/source/serializer.py

```diff
@@ -398,31 +370,30 @@ def validate_hosts(hosts):
 
         normalized_hosts = []
         host_errors = []
-        for host in hosts:
+        for host in hosts_list:
```

Comment:
think you could have just done `for host_range in hosts_list:`, since technically the string can still be a range of hosts. But no big deal.

Reply:
Agree.